demo/vl-rich-data-table.html

```diff
@@ -748,23 +748,33 @@ <h2>Rich data table met search filter</h2>
                                 "medewerkers": [{ "firstName": "Pieter", "lastName": "Beckers" }]
                             }]
                         };
-    
-                        element.addEventListener('change', (e) =&gt; {
+
+                        element.addEventListener(&apos;change&apos;, (e) =&gt; {
                             let newData = data.data;
+                            let totalItems = data.data.length;
+
+                            let filterEntries = undefined;
                             if (e.detail.formData) {
+                                filterEntries = [];
                                 for (let entry of e.detail.formData.entries()) {
                                     newData = filter(newData, entry[0], entry[1]);
+                                    totalItems = newData.length;
+                                    filterEntries.push({
+                                        name: entry[0],
+                                        value: entry[1]
+                                    });
                                 }
                             }
-    
+
+                            let sorting;
                             if (e.detail.sorting) {
-                                const sorting = e.detail.sorting;
+                                sorting = e.detail.sorting;
                                 newData = [...newData].sort((firstEl, secondEl) =&gt; {
                                     for (let i = 0; i &lt; sorting.length; i++) {
                                         const criteria = sorting[i];
                                         const firstValue = firstEl[criteria.name];
                                         const secondValue = secondEl[criteria.name];
-                                        const isAscending = criteria.direction === 'asc';
+                                        const isAscending = criteria.direction === &apos;asc&apos;;
                                         if (firstValue &lt; secondValue) {
                                             return isAscending ? -1 : 1;
                                         } else if (firstValue &gt; secondValue) {
```

Comment:
geen library die we hier kunnen binnen trekken om dat wat makkelijker te schrijven? allez, tis ook geen ramp he.

Reply:
Je bedoelt de hele filter sort page logica? Even rondgekeken en enkel deze gevonden: https://github.com/moinism/faltu , misschien beter dan toch onze code behouden denk ik dan.

Reply:
ja ca va, tis OK